contracts/modules/v2/StakingPool.sol

```diff
@@ -209,10 +224,10 @@ contract StakingPool is IStakingPool, ERC721 {
 
     activeStake = _activeStake;
     rewardPerSecond = _rewardPerSecond;
-    accRewardPerPoolShare = _accRewardPerShare;
+    accNxmPerRewardsShare = _accNxmPerRewardsShare;
     stakeSharesSupply = _stakeSharesSupply;
     rewardsSharesSupply = _rewardsSharesSupply;
-    lastRewardUpdate = _lastRewardUpdate;
+    lastAccNxmUpdate = _lastAccNxmUpdate;
```

Comment:
There are two variables that are being used together: 

```
  // accumulated rewarded nxm per reward share
  uint public accNxmPerRewardsShare;

  // timestamp when accNxmPerRewardsShare was last updated
  uint public lastAccNxmUpdate;
```

The variable name was chosen purposefully to indicate that it stores the timestamp when `accNxmPerRewardsShare` was last updated.

Reply:
`accNxmPerRewardsShare` -> `rewardShareValueInNXM` (since it's just the value of a reward share in nxm)
`lastAccNxmUpdate` -> `lastRewardShareValueUpdate` (reflects your comment much better: timestamp when accNxmPerRewardsShare was last updated)